client/src/actions/rounds.ts

```diff
@@ -73,43 +82,78 @@ export const loadRound = (address: string) => async (dispatch: Dispatch) => {
     status: Status.LoadingApplicationsStartTime,
   });
 
-  const ast: BigNumber = await contract.applicationsStartTime();
-  const applicationsStartTime = ast.toNumber();
+  let applicationsStartTime;
+  try {
+    const ast: BigNumber = await contract.applicationsStartTime();
+    applicationsStartTime = ast.toNumber();
+  } catch (e) {
+    dispatch(loadingError(address, "error loading application start time"));
+    console.error(e);
+    return;
+  }
 
   dispatch({
     type: ROUNDS_LOADING_ROUND,
     address,
     status: Status.LoadingApplicationsEndTime,
   });
 
-  const aet: BigNumber = await contract.applicationsEndTime();
-  const applicationsEndTime = aet.toNumber();
+  let applicationsEndTime;
+  try {
+    const aet: BigNumber = await contract.applicationsEndTime();
```

Comment:
It looks like the round manager team has indexed a lot of this data. It might be worth fetching everything from there? https://github.com/gitcoinco/grants-round/blob/1e8623ee84cf747d60e0ab756c3c0432b5a99527/packages/round-manager/src/features/api/services/round.ts#L128

Reply:
yes it makes sense, I added this ticket for that https://github.com/gitcoinco/grant-hub/issues/188